graphql/schemas/Inventory/productType.graphql

```diff
@@ -14,6 +14,7 @@ input ProductTypeInput {
     name: String!
     description: String
     weight: Int!
+    source_id: String
```

Comment:
@FredPeal make source_id type mixed we don't know what we can encounter in the future integration